openreview/arr/webfield/seniorAreaChairsWebfield.js

```diff
@@ -0,0 +1,79 @@
+const startParam = `${replaceAreaChairName(domain.content.area_chairs_assignment_id?.value)},tail:{ac.profile.id}`
+const traverseParam = `${replaceReviewerName(domain.content.reviewers_assignment_id?.value)}`
+const editParam = `${replaceReviewerName(domain.content.reviewers_invite_assignment_id?.value)}`
+const browseInvitations = []
+const assignmentInvitation = domain.content.sac_paper_assignments?.value ? null : domain.content.senior_area_chairs_assignment_id?.value
+
+if (domain.content.reviewers_affinity_score_id?.value) {
+  browseInvitations.push(replaceReviewerName(domain.content.reviewers_affinity_score_id?.value))
+}
+if (domain.content.bid_name?.value) {
+  browseInvitations.push(replaceReviewerName(`${domain.content.reviewers_id?.value}/-/${domain.content.bid_name?.value}`))
+}
+if (domain.content.reviewers_custom_max_papers_id?.value) {
+  browseInvitations.push(replaceReviewerName(`${domain.content.reviewers_custom_max_papers_id?.value},head:ignore`))
+}
+
+const otherParams = `&hide=${replaceReviewerName(domain.content.reviewers_conflict_id?.value)}&maxColumns=2&version=2&referrer=[Senior Area Chair Console](/group?id=${entity.id})`
+
+const start = `${domain.content.senior_area_chairs_assignment_id?.value},tail:${user.profile.id}`
+const traverse = `${domain.content.area_chairs_assignment_id?.value}`
+const edit = `${domain.content.area_chairs_assignment_id?.value};${domain.content.area_chairs_assignment_id?.value.replace('Assignment', 'Invite_Assignment')}`
+const browse = [
+    `${domain.content.area_chairs_id?.value}/-/Agreggate_Score`,
+    domain.content.area_chairs_affinity_score_id?.value, 
+    `${domain.content.area_chairs_id?.value}/-/Research_Area`,
+    `${domain.content.area_chairs_custom_max_papers_id?.value},head:ignore`,
+    `${domain.content.area_chairs_id?.value}/-/Registered_Load,head:ignore`,
+    `${domain.content.area_chairs_id?.value}/-/Emergency_Load,head:ignore`,
+    `${domain.content.area_chairs_id?.value}/-/Emergency,head:ignore`,
+    `${domain.content.area_chairs_id?.value}/-/Emergency_Area,head:ignore`,
+    
+]
+const other = `hide=${domain.content.area_chairs_conflict_id?.value}&maxColumns=2&version=2&referrer=[Senior Area Chair Console](/group?id=${entity.id})`
+
+const edgeBrowserLink = `/edges/browse?start=${start}&traverse=${traverse}&edit=${edit}&browse=${browse.join(';')}&${other}`
+
+return {
+  component: 'SeniorAreaChairConsole',
+  version: 1,
+  properties: {
+    header: {
+      title: `${committee_name.replaceAll('_', ' ')} Console`,
+      instructions: `<p class="dark">This page provides information and status updates for the ${domain.content.subtitle?.value}. It will be regularly updated as the conference progresses, so please check back frequently.</p><br><strong>AE Assignment Browser:</strong><a id="edge_browser_url" href="` + edgeBrowserLink + `"> Modify AE Assignments</a>`
```

Comment:
I wonder if there is a way to know when the assignments are deployed to show the url because before that they will get an error.